ja-JP/index.html

```diff
@@ -0,0 +1,256 @@
+        href="https://download.servo.org">Servo nightly build ダウンロード</a><br /><br />
+      </div>
+
+      <div class='col-sm-12 col-lg-6'>
+        <div class='thumbnail text-center'>
+          <a href="/screenshot.png">
+            <img class='media-object' src='/screenshot.png' alt="servo.org rendered with Servo"/>
+          </a>
+          2016年7月3日にServoでレンダリングされたページ
+        </div>
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <div id='community'>
+
+          <p>
+            Mozilla Researchで創案したServoプロジェクトは
+            個人からMozillaや三星(サムソン)のような企業に到るまで全世界的なコミュニティにようって開発されています。
+          </p>
+
+          <p>
+            Servoは今Linux、 macOS、 Windows、 Androidをサポートしています。
+          </p>
+
+        </div>
+      </div>
+
+    </div>
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>始める</h2>
+        <h3>必須条件</h3>
+        <p>
+          <a href="https://github.com/servo/servo#setting-up-your-environment">GitHub ページ</a>で
+          特定システムにインストールする為に必要なものを見つける事ができます。
+        </p>
+
+        <h3>ビルド</h3>
+        <p>
+          ServoはRustパッケージマネジャーのCargoと一緒にビルドされています。
+          また、MozillaのMach道具を利用してビルド及びその他の作業を調整します。
+        </p>
+
+        <h4>一般的なビルド</h4>
+        <p>
+          Servoを開発モードにビルドすると開発に有用に使用する事ができます。
+          しかし、バイナリーがビルドされるのにかかる時間が長くかかる事ができます。
+        </p>
+
+        <pre><code>git clone https://github.com/servo/servo
+cd servo
+./mach build --dev
+./mach run tests/html/about-mozilla.html</code></pre>
+
+        <p>
+          ベンチマーキング、性能テストまたは実際の使用の為に
+          <code>--release</code> フラグを追加して最適化されたビルドを作って下さい。
+        </p>
+
+        <pre><code>./mach build --release
+./mach run --release tests/html/about-mozilla.html</code></pre>
+
+        <h3>実行</h3>
+        <p>
+          該当コマンドを利用してServoを実行する事ができます。<code>./mach run [url]</code>
+        </p>
+
+        <h4>コマンド オプション</h4>
+
+        <ul>
+          <li>
+            <code>-p INTERVAL</code> 
+            プロファイラーを活性化して毎INTERVAL秒にコンソールにダンプを出力します。 
+          </li>
+          <li>
+            <code>-s SIZE</code> ウィンドウのサイズを設定します。
+            デフォルト値は512です。
+          </li>
+          <li>
+            <code>-z</code> 全てのグラフィック出力を無効化します。
+            JS 実行やレイアウトテストに有用します。
+          </li>
+        </ul>
+
+      </div>
+    </div>
+
+    <div class='row'>
+      <div class='col-sm-12'>
+        <h2 class='header'>貢献</h2>
+
+        <p id='contributing'>
+          Servoプロジェクトは経験が豊富した開発者から新しい開発者、皆の貢献を壮麗します。
+
+          ページ下段のリークを通してさらに詳しい情報を得る事ができて、
+          Servoをさらに良いものにする為に、全世界で働いてる数百人の貢献者が一緒にしています。
+        </p>
+
+        <p>
+          Servoへの貢献や彼による依存性はGitHubのプルリクエストを通じて行われなければならないです。
+          各プルリクエストは核心貢献者達が(パッチ適用の権限がある人)メインツリーにパッチしたり、必要な変更事項に対するフィードバックを受けます。
+          全ての貢献は定められたこのプルリクエスト形式を従い、さらに、核心貢献者達も従わなければなりません。
+        </p>
+
+        <p>
+          とある問題を解決したっかたら、先にGitHub issueに意見を提示して下さい。
+          これは、同問題に対する貢献者達の重複した努力を防止する為のものです。
```

Comment:
I wonder if there's a more polite way of putting it. I'm not well-versed in Japanese _keigo_, but I feel like there are other ways of putting it instead of just adding _-kudasai_ and _-desu_.

Reply:
I’m sorry. I have a Japanese certificate (JLPT) but, I haven’t studied polite expression. It’s my mistake. If you don’t mind, could you tell me what you know polite expression? I only know -shiteorimasu. If you tell me, I will reference it and edit.

Reply:
> e.g. instead of saying "TCO is not supported in Rust", the author says "I'd like to have Rust guarantee TCO".

As a speaker using japanese as mother tongue, I feel it's not always natural in Japanese expression.
Japanese speaker use the later expression pattern with the meaning that they have some apologize emotion or you need to express it to your majesty (including your boss 😆 ). 

Generally,  especially in a technical document, it's no problem to saying "TCO is not supported in Rust" if you'd like to express about "TCO is not supported in Rust". Of course, you should use polite expression :)

Reply:
> とある問題を解決したっかたら、先にGitHub issueに意見を提示して下さい。
> これは、同問題に対する貢献者達の重複した努力を防止する為のものです。
	
As my feeling, it would be more better that:

> 何か問題を解決したい場合、まずGitHub Issueに問題・意見を登録してください。
> これにより同じ問題に対しての重複した努力を避けることができます。

Reply:
ありがとうございます。
I will correct it by referring to this. When you see it, Is there anything else need to fix?

Reply:
> ありがとうございます。

You're welcome!

> When you see it, Is there anything else need to fix?

Yes. I seem some phraseology are not popular for native speaker.
I'll review them but I may need some days. 
(Sorry, my private is busy now 😢 )

Reply:
Okay, If so, I'll be waiting until the final review is complete.